src/app/combo/combo.sample.html

```diff
@@ -39,7 +39,7 @@ <h5 class="sample-title">Reactive Form</h5>
                     </form>
                 </div>
                 <div class="input-row">
-                    <igx-combo #playgroundCombo class="input-container" [placeholder]="'Locations'"
+                    <igx-combo #playgroundCombo class="input-container" [placeholder]="'Locations'" [showSearchCaseIcon]="'true'"
```

Comment:
[showSearchCaseIcon]="'true'" should be passed as a boolean like [showSearchCaseIcon]='true'